go/netutil/conn_test.go

```diff
@@ -77,13 +73,9 @@ func TestReadTimeout(t *testing.T) {
 
 	select {
 	case err := <-c:
-		if err == nil {
-			t.Fatalf("Expected error, got nil")
-		}
+		assert.Error(t, err)
 
-		if !strings.HasSuffix(err.Error(), "i/o timeout") {
-			t.Errorf("Expected error timeout, got %s", err)
-		}
+		assert.True(t, strings.HasSuffix(err.Error(), "i/o timeout"), "Expected error timeout, got %s", err)
```

Comment:
`assert.ErrorContains` here (and other places where we are asserting on the contents of error strings)

Reply:
done.